README.md

```diff
@@ -14,14 +14,15 @@ Use the [`subxt-cli`](./cli) tool to download the metadata for your target runti
 ```bash
 cargo install subxt-cli
 ```
-2. Save the encoded metadata to a file:
+2. Set the URL you want to use, you may skip this step and the default `http://localhost:9933/` will be used:
+```
+export SUBXT_URL=<your endpoint>
+```
```

Comment:
I prefer the original doc format (in other words; show the simplest steps here and mention below using `--url` or the new `SUBXT_URL` env var to set the URL).